### PR TITLE
[FUSEQE-8632] openwire and mysql stability improvement

### PR DESCRIPTION
**Pull request by mkralik3**
//test: `@DB-custom-api-connector-DB or @integrations-amq-to-rest or @openwire or @integrations-db-to-db-mysql`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
Connection - CRUD | Create and delete 
Connection - CRUD | Kebab menu 
Connection - CRUD | Kebab menu edit and view 
Integration - AMQ to REST | Publish subscribe on topic 
Integration - DB to API | Create an integration with custom API connector as middle step 
Integration - DB to DB mysql | Read and create operations 
Integration - DB to DB | Read and update operations 
Integration - Openwire to Openwire | Publish subscribe on queue 
Integration info | Check error for integration with the same name 
Integration info | Check error for integration which has same name after update 
```

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
Connection - CRUD | Create and delete 
Connection - CRUD | Kebab menu 
Connection - CRUD | Kebab menu edit and view 
Integration - AMQ to REST | Publish subscribe on topic 
Integration - DB to API | Create an integration with custom API connector as middle step 
Integration - DB to DB mysql | Read and create operations 
Integration - DB to DB | Read and update operations 
Integration - Openwire to Openwire | Publish subscribe on queue 
Integration info | Check error for integration with the same name 
Integration info | Check error for integration which has same name after update 
```

File: utilities/src/main/java/io/syndesis/qe/resource/impl/AMQ.java [hunk 1]
```diff
@@ -37,7 +37,9 @@ public void deploy() {
             templateParams.put("MQ_PASSWORD", "topSecret");
 
             OpenShiftUtils.getInstance().templates().withName("syndesis-amq").delete();
-
+            TestUtils.waitFor(() -> OpenShiftUtils.getInstance().getTemplates().stream()
```
Comment (avano) [inline utilities/src/main/java/io/syndesis/qe/resource/impl/AMQ.java:40]: OpenshiftUtils.getInstance().templates().withName("syndesis-amq").get() == null

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - DB to DB mysql | Read and create operations 
```

**Comment by mkralik3** [comment]
retest this please

**Comment by mkralik3** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - DB to DB mysql | Read and create operations 
```

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**